Aquality.Appium.Mobile/tests/Aquality.Appium.Mobile.Tests/Samples/Android/NativeApp/ApiDemos/Screens/AndroidScreen.cs

```diff
@@ -22,7 +21,8 @@ protected AndroidScreen(By locator, string name) : base(locator, name)
         protected void StartActivity(string appPackage, string appActivity, bool stopApp = true)
         {
             AqualityServices.LocalizedLogger.Info("loc.application.android.activity.start", appPackage, appActivity);
```

Comment:
we should probably add a separate utility for such functionality in future, if we really need it